.github/workflows/_test_t5x.yaml

```diff
@@ -257,6 +261,31 @@ jobs:
           name: ${{ steps.meta.outputs.JOB_NAME }}
           path: output/*
 
+  test_status:
+    needs: [multi-gpu-multi-node, single-process-multi-device]
+    if: ( always() )
+    runs-on: ubuntu-22.04
+    outputs:
+      TEST_STATUS: ${{ steps.status.outputs.TEST_STATUS }}
+    
+    steps:
+      - name: Download artifacts
+        uses: actions/download-artifact@v3
+
+      - name: Check status of all tests
+        shell: bash -x -e {0}
+        id: status
+        run: |
+          EXIT_STATUSES="${GITHUB_RUN_ID}-*/*-status.json"
```

Comment:
Could this reuse `_publish_badge.yaml`?

Reply:
What about this: we can reuse `_publish_badge.yaml` in `_test_*.yaml` and have `_publish_badge.yaml` output the "status". We can still keep _publish_badge.yaml's usage in `nightly-*.yaml`, and in the future that can be updated to eventually use the status output of each `_test_*.yaml`